org.firo.firo-qt.json

```diff
@@ -0,0 +1,73 @@
+{
+    "id": "org.firo.firo-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "23.08",
+    "command": "firo-qt",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=/run/tor/control.authcookie:ro"
+    ],
+    "modules": [
+        {
+            "name": "firo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -m 0744 -D -t /app/bin/internal bin/firod",
+                "install -m 0744 -D -t /app/bin/internal bin/firo-qt",
+                "install -m 0744 -D -t /app/bin/internal bin/firo-cli"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/firoorg/firo/releases/download/v0.14.13.1/firo-0.14.13.1-linux64.tar.gz",
+                    "sha256": "a6e763399cbe58ece0485232f44d1bedb9951995df437e005af3ef8a2a16aeba",
+                    "x-checker-data": {
+                        "type": "html",
+                        "url": "https://github.com/firoorg/firo/releases.atom",
+                        "version-pattern": "<title>Firo ([\\d\\.]+)</title>",
+                        "url-template": "https://github.com/firoorg/firo/releases/download/$version/firo-$version-linux64.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "misc",
```

Comment:
no reason to make this a separate module.

Reply:
moved everything to one single module.